packages/protocol/src/types.ts

```diff
@@ -478,7 +484,7 @@ export enum OrderStatusEnum {
 }
 
 /**
- * A Close can be sent by Alice or the PFI as a reply to an RFQ or a Quote
+ * A Close can only be sent by the PFI at any point during the exchange, to indicate the PFI will not continue the exchange.
```

Comment:
this implies the exchange is not completed, but it could have completed successfully

```suggestion
 * A Close, which only the PFI can send, can be submitted at any point during the exchange to indicate that the transaction has reached a terminal state.
```

Reply:
Taken your suggestion. FYI, the language was copied from spec:

https://github.com/TBD54566975/tbdex/tree/main/specs/protocol#close